conans/client/conanfile/implementations.py

```diff
@@ -0,0 +1,43 @@
+import inspect
+
+from conans.model.pkg_type import PackageType
+
+
+def call(module, implementation_name, conanfile_method, *args, **kwargs):
+    # first check that we have at least one function in the module that starts
+    # with the implementation name, then, when calling to the function name composed
+    # with the conanfile_method name we pass on the exception
+    def _has_function_starting_with(module, prefix):
```

Comment:
Maybe we want this to be checked earlier

Reply:
Yes, seems a bit too much of extra checking, for every recipe loading in the graph (hundreds of them), this will be repeated over and over.

I am fine with no error checking for the initial implementation